packages/nimble-components/src/tooltip/tests/tooltip-matrix.stories.ts

```diff
@@ -72,6 +72,8 @@ const component = (
             border: 1px solid var(${borderColor.cssCustomProperty});
             font: var(${bodyFont.cssCustomProperty});
             color: var(${bodyFontColor.cssCustomProperty});
+            width: 80px;
+            height: 60px;
```

Comment:
Looking at the chromatic ui tests these background colors on the tooltip don't look right:
![image](https://user-images.githubusercontent.com/1588923/224181579-23a68147-41c0-4019-97a9-a95ca098b294.png)

Reply:
I guess it had always been that way. I changed it so that the background for all three modes (in light theme) is white.

Reply:
Did you consult with Brandon before changing the background color. From what I can tell, the background of the default tooltip in the light theme should be Black15, as currently implemented in main.

https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/044414d7-1714-40f2-9679-2ce2c8202d1c/specs/

Reply:
Apparently it was correct the way it was, so I'm reverting my change to make them all match.